contributing/core_team.rst

```diff
@@ -205,6 +234,79 @@ following these rules:
 Getting the right category is important as it is used by automated tools to
 generate the CHANGELOG files when releasing new versions.
 
+.. tip::
+
+    Core team members are granted write-access to the main repository. To
+    avoid unintentional pushes to the main project (which in turn creates
+    new versions on Packagist), core team members are encouraged to have
+    two clones of the project locally:
+
+    #. A clone for your own contributions, which you use to push to your
+       fork on GitHub. Clear out the push URL for the main repository using
+       ``git remote set-url --push origin dev://null``;
+    #. A clone for merging, which you use in combination with ``gh`` and
+       allows you to push to the main repository.
+
+Upmerging Version Branches
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To synchronise changes in all versions, version branches are regularly
+merged from oldest to latest, called 'upmerging'. This is a manual process.
+There is no strict policy on when this occurs, but usually not more than
+once a day and at least once before monthly releases.
+
+Before starting the upmerge, Git must be configured to provide a merge
+summary by running:
+
+.. code-block:: terminal
+
+    $ git config --global merge.summary true
```

Comment:
Given https://git-scm.com/docs/git-merge#Documentation/git-merge.txt---summary and https://git-scm.com/docs/merge-config, I think it must be `merge.stat` instead:
```suggestion
    $ git config --global merge.stat true
```